pom.xml

```diff
@@ -204,7 +210,7 @@
         <dependency>
             <groupId>com.github.SkriptLang</groupId>
             <artifactId>Skript</artifactId>
-            <version>2.10.1</version>
+            <version>2.11.1</version>
```

Comment:
i know that there were some deprecated methods marked for removal, which I used in Skript... doesn't this break them?

Reply:
nvm, found out it works, but I'll have to fix that sooner or later anyway